website/content/docs/external_plugins/_index.md

```diff
@@ -4,133 +4,136 @@ title= "External Plugins"
+* `slash_replacer`. This overrides the default slash replacer `#`.
 * `state`. This corresponds to the `<state>` parameter in the plugin script's usage.
 
 Below is an example JSON object showcasing all possible keys at once.
 
 ```
 {
   "name": "some_entry",
-  "supported_actions": [
+  "methods": [
     "list"
   ],
   "cache_ttls": {
     "list": 30
   },
   "attributes": {
-    "atime": 1551942012,
     "mtime": 1551942012,
-    "ctime": 1551942012,
-    "mode": 511,
-    "size": 15600
+    "meta": {
+      "LastModifiedTime": 1551942012,
+      "Owner": "Wash",
+    }
   },
-  "state": "{\"klass\":\"AWS::Profile\"}"
+  "slash_replacer": ":",
+  "state": "{\"klass\":\"SSHFS::Directory\"}"
 }
 ```
 
-We see from `cache_ttls` that the result of `some_entry`'s `list` action will be cached for 30 seconds. We see from `attributes` that `some_entry` has some filesystem attributes that are defined (note that the mode is `0777`; `511` is its Base-10 representation). Finally, we see from `state` that `some_entry` has some state that Wash will pass-back in via the `<state>` parameter whenever it invokes one of its supported actions. In this case, only `list` is supported, and `<state>` is a stringified JSON object containing the entry's class (`AWS::Profile`) in whatever language the plugin script was written in.
+We see from `cache_ttls` that the result of `some_entry`'s `list` method will be cached for 30 seconds. We see from `attributes` that `some_entry` has an `mtime` attribute, and that it also includes the `meta` attribute. We see from `slash_replacer` that any `/`'es in the entry's returned name will be replaced by a `:` instead of a `#`. Finally, we see from `state` that `some_entry` has some state that Wash will pass-back in via the `<state>` parameter whenever it invokes one of its methods. In this case, only `list` is implemented, and `<state>` is a stringified JSON object containing the entry's class (`SSHFS::Directory`) in whatever language the plugin script was written in.
 
-The `init` action adopts the standard error conventions described in the [Errors](#errors) section.
+`init` adopts the standard error conventions described in the [Errors](#errors) section.
 
 ## list
-The `list` action is invoked as `<plugin_script> list <path> <state>`. When `list` is invoked, the script must output an array of JSON objects. Each JSON object has the same schema as the JSON object described in the `init` section (hence, `list` outputs an array of entries). Below is an example of valid output from the `list` action.
+`list` is invoked as `<plugin_script> list <path> <state>`. When `list` is invoked, the script must output an array of JSON objects. Each JSON object has the same schema as the JSON object described in the `init` section (hence, `list` outputs an array of entries). Below is an example of valid `list` output:
```

Comment:
One thing that's not really clear here is what path I should expect. Some examples might help.